website/views/shared/avatar/appearance.jade

```diff
@@ -15,6 +15,9 @@ mixin avatar(opts)
     each klass, item in visualBuffs
       span(ng-if='profile.stats.buffs.#{item}', class='#{klass}')
 
+    // Show flower if using Flower visual buff
+    span(ng-if='profile.stats.buffs.shinySeed', class='hair_flower_{{profile.preferences.hair.flower}}')
```

Comment:
The snowman has an earring and the ghost is holding a jewel! :) The starfish is a little confused, I'll grant you that, but the other two are definitely cute. I think we should go with that. People can always turn off the flower if they don't like it.